api/controller/caasapplicationprovisioner/client.go

```diff
@@ -86,14 +85,20 @@ func maybeNotFound(err *params.Error) error {
 	return errors.NewNotFound(err, "")
 }
 
-// Life returns the lifecycle state for the specified CAAS application
+// Life returns the lifecycle state for the specified application
```

Comment:
no longer an 'application' according to your switch statement

Reply:
Actually, the comment was:

```
// Life returns the lifecycle state for the specified CAAS application
// or unit in the current model.
```

The "or unit" was there (presumably a copy'n'paste error from earlier?) even though the function didn't actually work on units. Now it does. I just removed the "CAAS" part as it wasn't actually CAAS-specific.